src/loader/make-ctx.ts

```diff
@@ -69,7 +69,7 @@ function createTranslate (vm?: VM) {
     };
 
     let storedTranslations = {};
-    translate.setup = (newTranslations: Message | {} | null) => {
+    translate.setup = (newTranslations: Message | object | null) => {
```

Comment:
does it make any difference?

Reply:
eslint: avoid `{}` where possible -- `{}` actually means `any non-nullish value`.